src/App_old.js

```diff
@@ -0,0 +1,123 @@
+import { Component } from 'react';
```

Comment:
if it's old, then please remove from the git

Reply:
Ok, I removed it.